settings.py

```diff
@@ -69,11 +69,12 @@
 
 CSP_INCLUDE_NONCE_IN = ['script-src', 'style-src-elem']
 CSP_FONT_SRC = ('https://fonts.gstatic.com', 'data:')
-CSP_CONNECT_SRC = ("'self'", 'https://gtexportal.org', 'https://storage.googleapis.com') # used by IGV
+CSP_CONNECT_SRC = ("'self'", 'https://gtexportal.org', 'https://storage.googleapis.com', 'https://igv.org', 'https://data.broadinstitute.org') # used by IGV
```

Comment:
do we still need these new connect sources? I don't see them used anywhere so wanted to double check

Reply:
The `igv.org` must be still required. If we don't add it, there will be an error when opening an igv: `Refused to connect to 'https://igv.org/genomes/genomes.json?randomSeed=0.zx0ftapqh3r&version=2.8.6' because it violates the following Content Security Policy directive: "connect-src 'self' https://gtexportal.org https://storage.googleapis.com".`

The `data.broadinstitute.org` could be removed. It seems that it is for increasing an igv js use counter on a Broad's server. If it is refused to connect, it will generate an error but the error can be ignored. The error is `Refused to connect to 'https://data.broadinstitute.org/igv/projects/current/counter_igvjs.php?version=0' because it violates the following Content Security Policy directive: "connect-src 'self' https://gtexportal.org https://storage.googleapis.com".`

Reply:
But *why* are we trying to connect to `https://igv.org/genomes/genomes.json` in the first place? It specifies a bunch of default configurations so  you can initialize IGV with a genome version and it will know what configuration to use, but we use our own reference configuration so we shouldn't need to fetch the defaults at all. Maybe look in to the `loadDefaultGenomes` setting? Or look through the code to see if we actually need to fetch this?

And since it is just a counter, we should have the `data.broadinstitute.org` requests fail, the error is fine to ignore.